x/profiles/types/genesis.go

```diff
@@ -1,33 +1,27 @@
 package types
 
-import (
-	"fmt"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
 // GenesisState contains the data of the genesis state for the profile module
 type GenesisState struct {
-	Profiles           []Profile                   `json:"profiles" yaml:"profiles"`
-	Params             Params                      `json:"params" yaml:"params"`
-	UsersRelationships map[string][]sdk.AccAddress `json:"users_relationships"`
+	Profiles             []Profile             `json:"profiles" yaml:"profiles"`
+	Params               Params                `json:"params" yaml:"params"`
+	DTagTransferRequests []DTagTransferRequest `json:"dtag_transfer_requests" yaml:"dtag_transfer_requests"`
```

Comment:
Why was the `UsersRelationship` deleted from the genesis file?

Reply:
That field was an unused one that was left there forgotten after I moved `relationships` into a standalone module.